co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureMessageAmbassador.java

```diff
@@ -162,16 +164,17 @@ private synchronized void receiveInteraction(InfrastructureV2xMessageReception i
      * 
      *            Note: This function should be called after the
      *            onRsuRegistrationRequest
+     * @throws UnknownHostException
      */
-    private void onDsrcRegistrationRequest(String infrastructureId) {
+    private void onDsrcRegistrationRequest(String infrastructureId) throws UnknownHostException {
         // Create an InterfaceConfiguration object to represent the configuration of the
         // Ad-Hoc interface
         // Set the IP address and subnet mask to null for now
         // Set the transmit power to 50 dBm and the maximum range to 100 meters
         // NOTE: TODO Setup the IP address and subnet
         InterfaceConfiguration interfaceConfig = new InterfaceConfiguration.Builder(AdHocChannel.SCH1)
-                .ip(null)
-                .subnet(null)
+                .ip( (Inet4Address) Inet4Address.getByName("192.168.0.1") ) //TODO
```

Comment:
Can you explain what the TODO here is? Generally leaving notes for what needs to be done is good, in case someone else ends up picking up where you left off so they have context.